.gitignore

```diff
@@ -17,6 +17,7 @@ app/.pnp.js
 
 # testing
 app/coverage
+app/src/AdminPage/VisitorsTable/visitorPackage.json
```

Comment:
why do you want to git ignore this? (more a question for me to understand I suppose...!)